Packs/AWS/ReleaseNotes/2_1_5.md

```diff
@@ -8,4 +8,8 @@
 - Added support for the ***aws-ec2-instances-describe*** command that describes specified instances or all instances.
 - Added support for the ***aws-ec2-instances-run*** command that launches a specified number of instances using an AMI you have access to.
 - Added support for the ***aws-ec2-instances-terminate*** command that shuts down specified instances.
+- Added support for the ***aws-s3-bucket-encryption-get*** command that retrieves the default encryption configuration for an Amazon S3 bucket. Shows the server-side encryption settings applied to new objects stored in the bucket.
```

Comment:
Why is this not in a new file? And the pack version is not bumped?

Reply:
It will be merged as a single commit anyway, as this PR is merged to dev-cloud and not master